### PR TITLE
Add missing examples for Update operations and new definition for OperationLog Readonly contract

### PR DESCRIPTION
**Pull request by solankisamir**
Add examples to fix SDK Violations

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimtenant.json [hunk 1]
```diff
@@ -645,10 +660,34 @@
...
+        "action": {
+          "type": "string",
+          "description": "Action like create/update/delete."
+        },
+        "objectKey": {
+          "type": "string",
+          "description": "Identifier of the entity being created/updated/deleted."
+        }
+      },
+      "description": "Log of the entity being updated/created/updated."
```
Comment (marstr) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimtenant.json]: typo, "updated" is duplicated.

**Review by marstr — CHANGES_REQUESTED** [review]
Couple of typos, also could you include in the PR description/title that there is a new definition as well, not just added examples.

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/examples/ApiManagementListOperations.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+              "description": "Creates or updates the diagnostic setting for API Management service"
+            }
+          },
+          {
+            "name": "Microsoft.ApiManagement/service/tenant/operationResults/read",
+            "display": {
+              "provider": "Microsoft API Management",
+              "resource": "Results of async operations",
+              "operation": "Get operation results or Get operation result",
+              "description": "Get list of operation results or Get result of a specific operation"
```
[the remark this answers is not in the document]
Comment (solankisamir) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/examples/ApiManagementListOperations.json]: They were generated from real usage. But the number of operations were huge, I just truncated them.

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimtenant.json [hunk 3]
```diff
@@ -645,10 +660,34 @@
...
     },
+    "OperationResultLogItemContract": {
+      "properties": {
+        "objectType": {
+          "type": "string",
+          "description": "The type of entity contract."
+        },
+        "action": {
+          "type": "string",
+          "description": "Action like create/update/delete."
```
[the remark this answers is not in the document]
Comment (solankisamir) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimtenant.json]: No, this is an evolving list, these are read-only values

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimtenant.json (hunk 1 above)
Comment (solankisamir) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimtenant.json]: #fixed

**Review by marstr — APPROVED** [review]

**Comment by marstr** [comment]
Chatting with @solankisamir offline, he requested that @KedarJoshi review this before I merge.

**Comment by KedarJoshi** [comment]
#sign-off

**Pull request merged**